src/components/card.tsx

```diff
@@ -26,12 +27,25 @@ function Card({offer, classType, sizeImage}: CitiesCardProp): JSX.Element {
 
   const ratingInPercent = (rating / 5) * 100;
 
+  const mouseEnterHandler = () => offerStateHandler(offer);
```

Comment:
в реакте паттерн нейминга handle +
```suggestion
  const handleMouseEnter = () => offerStateHandler(offer);
```